code/modules/clothing/spacesuits/rig/modules/ds13/stasis.dm

```diff
@@ -0,0 +1,67 @@
+/obj/item/rig_module/mounted/stasis
+	name = "Stasis Module"
+	interface_name = "Stasis Module"
+	desc = "Stasis module is capable of producing a temporary time dilation, making objects move at an extremely slow rate for a period of time."
+	interface_desc = "Stasis module is capable of producing a temporary time dilation, making objects move at an extremely slow rate for a period of time."
+	icon = 'icons/obj/rig_modules.dmi'
+	icon_state = "module"
+
+	base_type = /obj/item/rig_module/mounted/stasis
+
+	selectable = 1
+	usable = 0
+	process_with_rig = FALSE
+
+	module_cooldown = 0
```

Comment:
this should probably have a cooldown

Reply:
in the game you can actually spam, I'll add few ticks cooldown